pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -185,8 +185,20 @@ type StorageClusterStatus struct {
 
 	// ExternalSecretHash holds the checksum value of external secret data.
 	ExternalSecretHash string `json:"externalSecretHash,omitempty"`
+
+	// Images holds the image reconclie status for all images reconciled by the operator
+	Images map[string]ComponentImageStatusRef `json:"imageReconcileState,omitempty"`
```

Comment:
Since the description shows up in the CSV along with the field, the JSON field name should match the Go field name. I like the name `Images` better, so please also update the JSON name to match.

Reply:
Sorry, my bad.
I have started with an ImageReconcileState than changed to Images but forgot to change the json filed name

Reply:
done